bzt/jmx/tools.py

```diff
@@ -437,6 +442,7 @@ def compile_scenario(self, scenario):
         elements.extend(self.__gen_keystore_config(scenario))
         elements.extend(self.__gen_data_sources(scenario))
         elements.extend(self.__gen_requests(scenario))
+        self.__add_jsr_elements(elements, scenario, False)
```

Comment:
Does the order of threadgroup (scenario) elements matter? If so, what is the best place for global jsr?